weblab/entities/processing.py

```diff
@@ -5,6 +5,7 @@
 from django.conf import settings
 from django.db import IntegrityError
 from django.urls import reverse
+from core.processing import prepend_callback_base
```

Comment:
The blank line should come before this import, not afterwards - it's part of our codebase not Django's.